google-services-plugin/build.gradle

```diff
@@ -1,29 +1,35 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2"
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.6' //1.8.4
```

Comment:
Not sure what this stray comment means.

Reply:
Just means there's a newer version available. It requires potentially breaking changes.

Reply:
Ack. Would you mind adding a full comment on when it would be appropriate to update to 1.8.4 ?

Reply:
I don't really know, it's really just whenever you feel like updating it to the latest. I've added the url for the release notes.